source/manual/vpn.html.md

```diff
@@ -31,7 +31,7 @@ If this VPN is down:
 
 Troubleshooting steps (aim is to switch off and on the VPN):
 
-1. Go to Production Skyscape portal, the credentials are in GOV.UK 2ndline Pass under: `ukcloud/portal`.
+1. Go to [Production Skyscape portal](https://www.skyscape.com/register/login.aspx), the credentials are in GOV.UK 2ndline Pass under: `ukcloud/portal`.
```

Comment:
Shouldn't this be https://portal.skyscapecloud.com/?

Reply:
It should indeed!